corehq/blobs/interface.py

```diff
@@ -74,3 +82,19 @@ def get_identifier(basename):
         else:
             prefix = "unsafe"
         return prefix + "." + uuid4().hex
+
+    @staticmethod
+    def get_args_for_delete(identifier=NOT_SET, bucket=NOT_SET):
+        if identifier is NOT_SET and bucket is NOT_SET:
+            raise ArgumentError("'identifier' and/or 'bucket' is required")
+        if identifier is None:
+            raise ArgumentError("refusing to delete entire bucket when "
+                                "null blob identifier is provided (either "
+                                "provide a real blob identifier or pass "
+                                "bucket as keyword argument)")
```

Comment:
If it were anything other than `None` it would not cause the [whole](https://github.com/dimagi/commcare-hq/blob/9dd9fd2a28851f6d094860454382172fd40ecc99/corehq/blobs/s3db.py#L62) [bucket](https://github.com/dimagi/commcare-hq/blob/9dd9fd2a28851f6d094860454382172fd40ecc99/corehq/blobs/fsdb.py#L54) to be deleted, so I think it's fine. Empty string would error due to missing blob, `False` would error on attempting to construct the blob path.